src/main/resources/l10n/JabRef_fr.properties

```diff
@@ -867,7 +867,7 @@ Please\ check\ your\ library\ file\ for\ wrong\ syntax.=Veuillez vérifier la sy
 SourceTab\ error=Erreur de SourceTab
 User\ input\ via\ entry-editor\ in\ `{}bibtex\ source`\ tab\ led\ to\ failure.=La saisie de l'utilisateur via l'éditeur d'entrée dans l'onglet `{}bibtex source` a conduit à un échec.
 
-Sort\ subgroups=Trier les sous-groupes
+Sort\ subgroups=Trier les sous-groupes A-Z
```

Comment:
Seems to got beyond the literal translation. Should we use this? @JabRef/vorstand

Reply:
Indeed, it does go beyond literal translation to make clear which sort order is used.
Bad idea or to be propagated?

Reply:
I think we then should better adjust the original string. E.g. sort subgroups alphabetically?

Reply:
The difficulties for me are:
- I am not sure how good my suggestions are, and it seems overkill to open a PR for this small thing.
- I find it difficult to find the file where is the initial string (other services, like https://hosted.weblate.org/ gives the source file, which is convenient).

Not: I used A-Z instead of "alphabétiquement" to keep the menu shorter.

Reply:
@ThiloteE  Should we replace that?

Reply:
I am not sure. Adjusting the original string also, intuitively seems better to me, but if the original string cannot be found ... i guess it's ok? Also, I don't know how it will look like in the UI

Reply:
@mlep  https://github.com/JabRef/jabref/blob/fe2ceef955227dd6b7b753a53310552ab12938dc/src/main/java/org/jabref/gui/groups/GroupTreeView.java#L432-L433 
here is the corresponding line
The en.properties needs to be adjusted then as well

Reply:
Fixed in https://github.com/JabRef/jabref/pull/9313